### ISSUE TITLE
list all your existing collaborators  when adding a collaborators 

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#2757]
Right now, the "add collaborator" dialog only shows a search box.

The goal of this ticket is to add a dropdown list for adding an existing collaborator of yours from another project you're part of. That way, you can select someone from your "known circle" of cocalc users instead of having to search for.

**Comment by williamstein** [comment]
I don't agree that we should implement this.  It's clutter.  Facebook doesn't need it.  By just showing all collabs first, we accomplish the same thing better.

counterargument?

**Comment by williamstein** [comment]
![screenshot 2018-03-16 at 11 09 42 am](https://user-images.githubusercontent.com/1276278/37537364-8bc99746-290a-11e8-9647-16592fecef6a.png)
